.github/workflows/release.yaml

```diff
@@ -3,30 +3,23 @@ name: VPC CNI Release
 on: [push, workflow_dispatch]
 
 env:
-  DEFAULT_GO_VERSION: ^1.18
   GITHUB_USERNAME: ${{ secrets.EKS_BOT_GITHUB_USERNAME }}
   GITHUB_TOKEN: ${{ secrets.EKS_BOT_GITHUB_TOKEN }}
 
 jobs:
-
   release:
-    name: Release
-    runs-on: ubuntu-20.04
     if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
```

Comment:
[minor] - ubuntu-latest could be used here as well.

Reply:
+1, nice catch

Reply:
addressed